app/editor/src/components/formik/textarea/styled/FormikTextArea.ts

```diff
@@ -1,3 +1,5 @@
 import styled from 'styled-components';
 
-export const FormikTextArea = styled.div``;
+export const FormikTextArea = styled.div`
+  width: 100%;
```

Comment:
Depending on other values this can cause overflow.  `max-content` may be a better value.